### DISCUSSION TITLE
Minimize work during key interaction-Timespan mode

### DISCUSSION THREAD

**Discussion by SivasubramanianV** [GoogleChrome/lighthouse#15427] [Q&A]
<img width="907" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/68318570/e8c8adbe-d92d-485f-a39c-a7371f383e30">
I saw this diagnostic when i was doing timespan testing of a page. That involved entering some text in a field on a page and navigating to another page in a SPA. I tried to find what this meant but I could not find anything relevant to this. What does this mean to INP. 
I also could see some event target and phase in this diagnostic. As a note, I am using webdriverio pupeteer automation script to get this.
Thanks

**Comment by adamraine** [comment]
This is the user interaction which is the INP event. 350ms should always correspond to your INP value. Work during the key interaction could have delayed the next paint, so this audit attempts to attribute the tasks which are running during the key interaction.

**Reply by SivasubramanianV** [reply]
@adamraine Thanks Adam..how does work during user interaction impact the next paint? If the same work happens when user is not interacting, will that not impact the next paint? Can you please explain it in more detail. Also any link that would explain this would help me.

**Reply by adamraine** [reply]
https://web.dev/optimize-inp/ may offer a better explanation.

cc @brendankenny
